data/json/monsters/zed_explosive.json

```diff
@@ -176,6 +176,38 @@
     "fungalize_into": "mon_zombie_gasbag_fungus",
     "flags": [ "SEES", "HEARS", "SMELLS", "STUMBLES", "WARM", "POISON", "NO_BREATHE", "REVIVES", "FILTHY" ]
   },
+  {
+    "id": "mon_zombie_relax_gasbag",
+    "type": "MONSTER",
+    "name": { "str": "sleepwalker zombie" },
+    "description": "A zombie swollen with a blueish, oddly sweet smelling gas. It stumbles around even more than regular zombies, as if it were heavily intoxicated.",
```

Comment:
```suggestion
    "description": "A zombie swollen with a bluish, oddly sweet smelling gas. It stumbles around even more than regular zombies, as if it was heavily intoxicated.",
```

Reply:
Maybe even just "as if heavily intoxicated"?

Reply:
done.

Reply:
I think you missed the "bluish" change. Seems like "blueish" is an accepted spelling, but "bluish" is definitely more common.

Reply:
True. But we have blueish nowhere currently but "bluish" gives me two hits with grep -r. Nonetheless a very small issue.